easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.5.3-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,244 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow'
+version = '2.5.3'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('Bazel', '3.7.2'),
+    ('protobuf', '3.17.3'),
+    # git 2.x required, see also https://github.com/tensorflow/tensorflow/issues/29053
+    ('git', '2.32.0', '-nodocs'),
+    ('pybind11', '2.6.2'),
```

Comment:
With `--review-pr` I saw that the `TensorFlow-2.5.0-fosscuda-2020b.eb` also has `pkg-config` here (see [here](https://github.com/easybuilders/easybuild-easyconfigs/blob/50885ea5e4ee9c44720c92b47fc14a17fa05d4c5/easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.5.0-fosscuda-2020b.eb#L18)). Any idea whether or not it should be here as well? (if so, that would also apply to the non-CUDA version of course)

Reply:
That build dependency on `pkgconfig` is only needed if `h5py` is bundled in the easyconfig of TensorFlow. In this case `h5py` is a regular dependency and the easyconfig of `h5py` already takes care of depending on `pkgconfig`.

https://github.com/easybuilders/easybuild-easyconfigs/blob/4eb9d9b2ff006c352f8150520cde479306591c98/easybuild/easyconfigs/h/h5py/h5py-3.2.1-foss-2021a.eb#L17

It's the same situation as the easyconfigs of TensorFlow 2.6 in this toolchain.

Reply:
Ah ok, clear, thanks! :)